packages/manager-components/src/components/atoms/description/description.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { waitFor, screen } from '@testing-library/react';
+import { render } from '../../../utils/test.provider';
+import { Description, DescriptionProps } from './description.component';
+import { description } from './description.stories';
+import '@testing-library/jest-dom';
+
+const setupSpecTest = async (customProps?: Partial<DescriptionProps>) =>
+  waitFor(() => render(<Description {...description} {...customProps} />));
+
+describe('description component', () => {
+  it('renders description correctly', async () => {
+    await setupSpecTest();
+
+    const descriptionText = screen.getByText('Lorem ipsum');
```

Comment:
So if you change the default text of the story, it breaks the test

I don't think it's a good idea to couple the story and the test